web/react/components/popover_list_members.jsx

```diff
@@ -32,22 +34,28 @@ export default class PopoverListMembers extends React.Component {
     }
     render() {
         let popoverHtml = '';
+        let count = '-';
         const members = this.props.members;
-        let count;
-        if (members.length > 20) {
-            count = '20+';
-        } else {
-            count = members.length || '-';
-        }
+        const teamMembers = UserStore.getProfilesUsernameMap();
 
-        if (members) {
+        if (members && teamMembers) {
             members.sort(function compareByLocal(a, b) {
                 return a.username.localeCompare(b.username);
             });
 
+            count = 0;
             members.forEach(function addMemberElement(m) {
-                popoverHtml += `<div class='text--nowrap'>${m.username}</div>`;
+                if (teamMembers[m.username] && teamMembers[m.username].delete_at <= 0) {
+                    popoverHtml += `<div class='text--nowrap'>${m.username}</div>`;
+                    count++;
+                }
             });
+
+            if (count > 20) {
+                count = '20+';
```

Comment:
I can't believe ESLint isn't failing on this. `count` is originally a string and then becomes an int and then becomes a string again here.

I think you should use two separate variables here, an int for counting and a string for the final representation of the count